src/components/link/link.ts

```diff
@@ -31,17 +23,11 @@ const ICON_NAME_DEFAULT: string = 'm-svg__chevron--right';
 @WithRender
 @Component
 export class MLink extends ModulVue {
-    @Prop({ default: '/' })
-    public url: string;
+    @Prop()
+    public url: string | Object;
 
```

Comment:
Idéalement ca serait mieux de toujours  utiliser des type primitif pour les props (string boolean number ) Car les object ne sont pas réactif a moins d'être immutable ce qui cause plein de problème dur a corriger

si on doit utiliser un Object on doit le typé comme suit:

example: 

```
interface MAccordionTransitionProps {
    heightDelta?: number;
    transition?: boolean;
}
```

Reply:
Le type **Object** a été utilisé dans ce cas, car la prop `url` est utilisé comme valeur de la prop  `to`  du composant vue `<router-link>`. La prop `to` accept les `Object`

Reply:
ah ok tu fais reference à https://router.vuejs.org/api/#to donc ce que tu veux exposé c'est Location. C'est important de garder les types

ie  

``
public url: string | Location;
``